CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.1.2] - 2022-12-13
+
+### Fixed
+
+- Update `stomp.py` from `7.0.0` to `~=8.1.0`.
```

Comment:
[fix](https://www.conventionalcommits.org/en/v1.0.0/) is used for errors, in which case it would be a feat. What do you think?

Reply:
Chaged to `Changed` following the [link](https://keepachangelog.com/en/1.0.0/)